### DISCUSSION TITLE
Allow link more than 10 pull requests from an issue

### DISCUSSION THREAD

**Discussion by cilerler** [community/community#32870] [Projects and Issues]
As a developer, I would like to link as many pull-requests I can link so I can manage multi-repository updates easily.

With [the latest update](https://github.blog/changelog/2022-09-07-link-existing-branches-to-an-issue/), we are able to link cross repository pull requests to an issue.  However, there is a 10 links limit, which is not enough when you have 20+ micro-services.

**Comment by bogatkaj** [comment]
Also not enought when you need to merge a large update from the dev-branch to the release-branch

**Comment by hrmnjt** [comment]
I've a practice of creating a large issue and address it with multiple PRs. 
Writing `fixes #PRnumber` to the parent issue links the PR to issue but on the issues page i.e. https://github.com/ORG/REPO/issues it shows count 10 for linked PRs which is incorrect information. If it is not possible to show exact number of linked PRs, it would be great to indicate 10+ at least.

example
<img width="1457" alt="image" src="https://github.com/user-attachments/assets/de44e317-16f8-4a4f-b304-852c7973b5f6">
I have 14 PRs linked for this issue

**Comment by VECKAKALENDAR** [comment]
For those looking into allowing more than 10 pull requests from an issue (#32870), having clear guidelines is crucial. Similarly, staying informed about financial trends is important. For up-to-date information on gold prices, visit our page on[ gold prices in Egypt](https://goldpricesegypt.com/). It provides the latest updates and insights into the precious metal market.

**Comment by tino** [comment]
Is this still on the roadmap? There seems to be no way to find out which PRs belong to an issue (or a milestone, through issues), neither through the API nor through gh.

**Comment by Itschotsch** [comment]
I also see "You may only link 10 issues." What is the reason behind this artificial limit of 10 linked issues? Any way to disable it?
